README.md

```diff
@@ -46,7 +36,7 @@ It is strongly advised to install as much as possible prior to the workshop.
  * JDK 11 installed
  * Maven >= 3.6.3 advised, possibly with JAVA_HOME configured appropriately
  * A favorite HTTP client like curl
- * Docker >= 1.13.1 installed (if docker is missing, that's no big deal, you may only miss part-3 and part-7)
+ * Docker >= 1.13.1 installed (if docker is missing, that's no big deal, you may only miss part-5 and some concepts in part-5)
```

Comment:
Not sure about this part. After the renaming part-5 is kafka and part-7 is native-mode.
Is the intent to say "you may only miss part-7 and some concepts in part-5" ?